packages/beachball/src/changelog/writeChangelog.ts

```diff
@@ -105,17 +124,31 @@ function writeChangelogFiles(changelog: PackageChangelog, changelogPath: string,
     const changelogFile = path.join(changelogPath, 'CHANGELOG.md');
     const previousContent = fs.existsSync(changelogFile) ? fs.readFileSync(changelogFile).toString() : '';
 
-    const nextContent = renderChangelog(previousContent, changelog, isGroupedChangelog);
-    fs.writeFileSync(changelogFile, nextContent);
-  }
-  try {
-    const changelogJsonFile = path.join(changelogPath, 'CHANGELOG.json');
-    const previousJson = fs.existsSync(changelogJsonFile)
-      ? JSON.parse(fs.readFileSync(changelogJsonFile).toString())
-      : { entries: [] };
-    const nextJson = renderJsonChangelog(previousJson, changelog);
-    fs.writeFileSync(changelogJsonFile, JSON.stringify(nextJson, null, 2));
-  } catch (e) {
-    console.warn('The CHANGELOG.json file is invalid, skipping writing to it', e);
+    const renderCustomPackageChangelog = options.changelog?.renderPackageChangelog;
+    let renderPackageChangelog;
+    if (renderCustomPackageChangelog) {
+      renderPackageChangelog = () => {
```

Comment:
It would be nice if we could figure out an API for letting people do *some* modifications to the changelog without having to reinvent all the logic. Not sure exactly what that would look like though. Some ways it could be broken down:
- custom entry renderer
- custom section header renderer
- plus a full custom renderer option

Reply:
if need, i would give back default renderers for different sections: header/entry header/entry body. but they are not helpful for our use case right now. so i'd like to avoid until there is a need

Reply:
I think actually it would be helpful now with your PR to Fabric--that way we don't have to entirely reinvent the wheel when all we want to customize is the section header so far.

Reply:
originally i was thinking giving `renderChangelogEntries` as default renderer, but the moment you add PR link, you can't use it any more

Reply:
Right, that's why I think we should have multiple custom renderers. It's a little more complex but not terrible.